MKL_README.md

```diff
@@ -17,3 +17,106 @@ Installing and enabling the full MKL installation enables MKL support for all op
 
   5. Run 'sudo python setup.py install'
 
+
+## Build/Install MXNet with a full MKL installation on Windows:
+
+To build and install MXNet yourself, you need the following dependencies. Install the required dependencies:
+
+1. If [Microsoft Visual Studio 2015](https://www.visualstudio.com/vs/older-downloads/) is not already installed, download and install it. You can download and install the free community edition.
+2. Download and Install [CMake](https://cmake.org/) if it is not already installed.
+3. Download and install [OpenCV](http://sourceforge.net/projects/opencvlibrary/files/opencv-win/3.0.0/opencv-3.0.0.exe/download).
+4. Unzip the OpenCV package.
+5. Set the environment variable ```OpenCV_DIR``` to point to the ```OpenCV build directory``` (```C:\opencv\build\x64\vc14``` for example). Also, you need to add the OpenCV bin directory (```C:\opencv\build\x64\vc14\bin``` for example) to the ``PATH`` variable.
+6. If you have Intel Math Kernel Library (MKL) installed, set ```MKL_ROOT``` to point to ```MKL``` directory that contains the ```include``` and ```lib```. Typically, you can find the directory in
+```C:\Program Files (x86)\IntelSWTools\compilers_and_libraries_2018\windows\mkl```.
+7. If you don't have the Intel Math Kernel Library (MKL) installed, download and install [OpenBlas](http://sourceforge.net/projects/openblas/files/v0.2.14/). Note that you should also download ```mingw64.dll.zip`` along with openBLAS and add them to PATH.
+8. Set the environment variable ```OpenBLAS_HOME``` to point to the ```OpenBLAS``` directory that contains the ```include``` and ```lib``` directories. Typically, you can find the directory in ```C:\Program files (x86)\OpenBLAS\```. 
+
+After you have installed all of the required dependencies, build the MXNet source code:
+
+1. Download the MXNet source code from [GitHub](https://github.com/dmlc/mxnet). Don't forget to pull the submodules:
+```
+    git clone https://github.com/apache/incubator-mxnet.git ~/mxnet --recursive
+```
+
+2. Update mkldnn to the newest:
+```
+    cd 3rdparty/mkldnn/ && git checkout master && git pull
+```
```

Comment:
This is because mxnet doesn't point to the latest commit in mkldnn master branch, right now? 
This is indeed awkward. Ideally, we don't want users to do it manually. but on the other hand, it's unclear if we should move to the latest commit in mkldnn repo right now.

Reply:
I have to agree with @zheng-da would refrain from adding this instruction. We should only support the versions which are part of our repository. If there's a need to upgrade, we'll have to do it here. Otherwise we're going to run into issues with people who upgrade on their own while we have not validated that version with our CI.